core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.ts

```diff
@@ -332,10 +383,59 @@ angular.module('oppia').directive('stateTranslation', [
             return {'border-left': '3px solid ' + color};
           };
 
-          $scope.getHtmlSummary = function(subtitledHtml) {
-            var htmlAsPlainText = $filter(
-              'formatRtePreview')(subtitledHtml.getHtml());
-            return htmlAsPlainText;
+          $scope.getSubtitledContentSummary = function(subtitledContent) {
+            if (subtitledContent instanceof SubtitledHtml) {
+              return $filter('formatRtePreview')(subtitledContent.getHtml());
+            } else if (subtitledContent instanceof SubtitledUnicode) {
+              return subtitledContent.getUnicode();
+            }
+          };
+
+          $scope.getInteractionCustomizationArgTranslatableContent = function(
+              customizationArgs: InteractionCustomizationArgs
+          ): {name: string, content: SubtitledUnicode|SubtitledHtml}[] {
+            const translatableContent = [];
+
+            const camelCaseToSentenceCase = (s) => {
+              // Add a space in front of capital letters.
+              s = s.replace(/([A-Z])/g, ' $1');
+              // Captialize first letter.
+              s = s.charAt(0).toUpperCase() + s.slice(1);
+              return s;
+            };
+
+            const traverseValueAndRetrieveSubtitled = (
```

Comment:
...RetriveSubtitles or ...RetriveSubitledContent? (The ending with subtitled seems incomplete!)

Reply:
Done